auth-web/src/components/auth/account-settings/payment/AccountPaymentMethods.vue

```diff
@@ -473,11 +476,13 @@ export default defineComponent({
           state.errorTitle = 'Error'
           switch (error.response.status) {
             case 409:
-              state.errorText = error.response.data.message?.detail || error.response.data.message
-              break
             case 400:
-              state.errorText = error.response.data.message?.detail || error.response.data.message
-              state.errorTitle = error.response.data.message?.title || 'Error'
+              state.errorText = `${formatText(error.response.data.code)}<br>` +
```

Comment:
What about account create? I think this same code is used around there

Maybe we should refactor this error handling?